### PR TITLE
creating new reporting view

### PR DESCRIPTION
**Pull request by cristinabaldor**
# Pull Request

## Summary & Motivation

edits to staging table date fields, replacing int table with reporting view

[//]: # "When merged, this pull request will..."

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

- [ ] Ensure you are using the `union_dataset_join_clause()` macro for queries that employ any
      models from these datasets:
  - deanslist
  - edplan
  - iready
  - pearson
  - powerschool
  - renlearn
  - titan

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207850166984949

### CODE REVIEW

File: src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql [hunk 1]
```diff
@@ -1,54 +1,42 @@
...
-    time_in_application_status_interview_phone_screen_requested,
-from {{ source("smartrecruiters", "src_smartrecruiters__applications") }}
+    a.*,
+
+    cast(new_date_timestamp as date) as new_date,
+    cast(phone_screen_complete_date_timestamp as date) as phone_screen_complete_date,
+    cast(phone_screen_requested_date_timestamp as date) as phone_screen_requested_date,
+    cast(demo_date_timestamp as date) as demo_date,
+    cast(offer_date_timestamp as date) as offer_date,
+    cast(hired_date_timestamp as date) as hired_date,
```
Comment (cbini) [inline src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql:41]: are these dates already before they're converted to timestamp?

Comment (cbini) [inline src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql:41]: are these dates already before they're converted to timestamp?

**Review by cbini — CHANGES_REQUESTED** [review]

File: src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql (hunk 1 above)
Comment (cristinabaldor) [reply src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql:41]: they come out of smart recruiters as these weird timestamps that I couldn't just convert to date for some reason, I kept getting errors

Comment (cbini) [reply src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql:41]: any idea what timezone these are in?

Comment (cbini) [reply src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql:41]: converting it directly to date might create a day difference if they're UTC

**Review by cbini — APPROVED** [review]

File: src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql (hunk 1 above)
Comment (cbini) [reply src/dbt/kipptaf/models/smartrecruiters/staging/stg_smartrecruiters__applications.sql]: gonna say this is fine for now, if you notice dates are off we can fix them later

**Pull request merged**